convert/lib/convert.py

```diff
@@ -0,0 +1,614 @@
+#!/usr/bin/env python3
+import argparse
+import boto3
+import json
+import re
+import yaml
+from copy import deepcopy
+from functools import reduce
+from pprint import pprint
+from subprocess import run
+from time import sleep
+
+from .meta import (
+    cdk_ids,
+    cf_status,
+    resource_template,
+    efs_backup_resources,
+    route53_resource,
+    stack_map,
+)
+from .nuke import nuke
+
+
+clean_categories = [x.name for x in cdk_ids if x.name != "cloudformation_stack"]
+
+
+class app:
+    def __init__(self):
+        self.parse_args()
+        self.args.command()
+
+    def get_stacks(self, stack: str = None, full: bool = False):
+        stacks = {"resources": {}}
+        for s in self.cf.describe_stack_resources(StackName=stack or self.stack_name)[
```

Comment:
This doesn't need to be paginated?

Reply:
Our stacks appear to have fewer than 100 resources, but I did mean to get back to this later. I've paginated it just in case.